dist/45e582c10309a2b2f5f0.js

```diff
@@ -1,22 +1,20 @@
 import './styles/style.css';
 import './styles/mystyle.css';
-import {
-  fetchFoodInfo, enterMeals, oneLike, fetchInfo, showInfo,
-} from './module/mealDetails.js';
 
-import { showPopup, togglePopup } from './module/popup.js';
+import fetchFoodInfo from './module/foodApi.js';
+import enterMeals from './module/foodCards.js';
+import { oneLike, fetchInfo, showInfo } from './module/like.js';
+import { mealsnumber, showMeals } from './module/mealCount';
 
 const foodDescr = await fetchFoodInfo();
 const metricsInfo = await fetchInfo();
-
 enterMeals(foodDescr);
+const opinionsNo = await mealsnumber();
+showMeals(opinionsNo);
 foodDescr.forEach((element) => {
   showInfo(metricsInfo, `M${element.idMeal}`);
 });
 
-const comment = [...document.querySelectorAll('.comment')];
-const popWindow = document.getElementById('my-popup-window');
-
```

Comment:
For the second time, you keep breaking my code. Here, you deleted this lines which is responsible for the popup functionality.

Reply:
Returned.